src/Integration.UnitTests/Exclusions/AnalyzableFileIndicatorTests.cs

```diff
@@ -65,8 +65,8 @@ public void ShouldAnalyze_NoExclusionSettings_True()
             bool globalExclusionsApply,
             bool expectedResult)
         {
-            var projectExclusions = new[] { "exclusion1", "exclusion2" };
-            var globalExclusions = new[] { "exclusion3", "exclusion4" };
+            var projectExclusions = new[] { "**/exclusion1", "**/exclusion2" };
```

Comment:
I really should've created IServerExclusions and mocked it, but I didn't think about it until later...